stream/stream_file.c

```diff
@@ -272,10 +272,14 @@ static int open_f(stream_t *stream, const struct stream_open_args *args)
     if (strncmp(url, "fd://", 5) == 0 || is_fdclose) {
         char *begin = strstr(stream->url, "://") + 3, *end = NULL;
         p->fd = strtol(begin, &end, 0);
-        if (!end || end == begin || end[0]) {
+        if (!end || end == begin) {
             MP_ERR(stream, "Invalid FD: %s\n", stream->url);
             return STREAM_ERROR;
         }
+        if (strncmp(end, "/p/", 3) == 0) {
```

Comment:
If you add this functionality you need to document it in the appropriate RST file. Hidden features are not fun.

Reply:
Thanks for the correction, I will add the corresponding instructions in the documentation.